template-parts/content-post.php

```diff
@@ -67,6 +67,13 @@
 					<h2 class="nhsuk-card__heading nhsuk-heading-m">
 						<a class="nhsuk-card__link" href="<?php the_permalink(); ?>">
 							<?php the_title(); ?>
+							<?php // Add date if event
+							if ( get_post_type() === 'tribe_events' ) {
+								$event_date = strtotime ( $post->start_date );
+								$event_date = date ( 'd M Y', $event_date );
+								echo '(' . $event_date . ')';
```

Comment:
I love this approach. Can we tweak it slightly?
`echo '<span class="nhsuk-u-visually-hidden">' . $event_date . '</span>';`
That way the appearance is the same for users, but screen readers will pick the date up

Reply:
Good idea. That's done.
![Screenshot 2021-01-21 at 16 02 33](https://user-images.githubusercontent.com/1991226/105376919-2d9ebd80-5c02-11eb-8c42-12bc10979be2.png)